_data/landscape/meshes.yml

```diff
@@ -389,6 +436,8 @@
   tracing: "?"
   encryption: "?"
   multi-tenant-score: 1
+  spiffe: "?"
```

Comment:
The project is working on exposing this externally https://www.youtube.com/watch?v=fxtG1wuLSDw&feature=youtu.be&t=17m37s